### PR TITLE
XW-1730 | added translations for local sitemap

### PR DESCRIPTION
**Pull request by Warkot**
@Wikia/x-wing

### CODE REVIEW

File: i18n/zh-hans/design-system.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
   "global-footer-wikia-company-overview-link-wikia-gives-back":"Wikia回馈",
   "global-footer-wikia-site-overview-link-api":"API",
   "global-footer-wikia-site-overview-link-global-sitemap":"全球网站地图",
+  "global-footer-wikia-site-overview-link-local-sitemap":"本地网站地图",
```
Comment (rogatty) [inline i18n/zh-hans/design-system.json:40]: Looks fine, I guess ¯_(ツ)_/¯

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
How about `nl`? do we have translation for that?
If not, can you put `en` there?

**Comment by Warkot** [comment]
No, `nl` is dropped from support for now.

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by Warkot** [comment]
I response to your question, @tomasznapieralski: https://github.com/Wikia/design-system-i18n/pull/13.

**Pull request merged**